src/FastState.cpp

```diff
@@ -27,6 +27,7 @@
 #include "Utils.h"
 #include "Zobrist.h"
 
+extern bool cfg_dyn_komi;
```

Comment:
Just include GTP.h.

Reply:
seems to work here, though I recall having encountered a cycle of includes that led to compilation error...